lib/workers/repository/finalise/repository-statistics.ts

```diff
@@ -33,3 +35,55 @@ export function runRenovateRepoStats(
   }
   logger.debug({ stats: prStats }, `Renovate repository PR statistics`);
 }
+
+interface BranchMetadata {
+  branchName: string;
+  sha: string | null;
+  parentSha: string | null;
+  automerge: boolean;
+  isModified: boolean;
+}
+
+interface BaseBranchMetadata {
+  branchName: string;
+  sha: string;
+}
+
+export function runBranchSummery(): void {
+  const { scan, branches } = getCache();
+
+  const baseMetadata: BaseBranchMetadata[] = [];
+  for (const [branchName, cached] of Object.entries(scan ?? {})) {
+    baseMetadata.push({ branchName, sha: cached.sha });
+  }
+
+  function unwrap({
+    branchName,
+    sha,
+    parentSha,
+    automerge,
+    isModified,
+  }: BranchCache): BranchMetadata {
+    return { branchName, sha, parentSha, automerge, isModified };
+  }
+
+  const branchMetadata: BranchMetadata[] = [];
+  const inactiveBranches: string[] = [];
+
+  for (const branch of branches ?? []) {
+    if (!branch.sha) {
```

Comment:
this looks to me more like if/else than if + continue..

maybe do it in 1 row? is that possible? something like this:
branch.sha ? branchMetadata.push(unwrap(branch)) : inactiveBranches.push(branch.branchName)

Reply:
or if/else is fine too for readability